### PR TITLE
Start with technote about renv

### PR DESCRIPTION
**Pull request by jeroen**

### CODE REVIEW

File: content/blog/2022-01-05-runiverse-renv/index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+---
+
+<small><em>This post is part of a series of technotes about [r-universe](https://r-universe.dev), a new umbrella project by rOpenSci under which we experiment with various ideas for improving publication and discovery of research software in R.
+As the project evolves, we will post updates to document features and technical details.
+For more information, visit the [r-universe project page](/r-universe/).</em></small>
+
+
+## What is renv
+
+RStudio's [renv](https://rstudio.github.io/renv/) package is a powerful dependency management toolkit for R. It allows you to create a *lockfile* that records the exact versions of R packages used in a given project, and provides tooling to install exactly those same versions on another machine, or at a later point in time. This is very useful to create an isolated environment for reproducibility or production purposes.
```
Comment (maelle) [inline content/blog/2022-01-05-runiverse-renv/index.md:21]: Could this sentence add something about "isolated library per project" or so

File: content/blog/2022-01-05-runiverse-renv/index.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+The latest version of renv (0.15.0) now supports restoring packages that were installed from r-universe. In this post we explain how this works, and why.
```
Comment (maelle) [inline content/blog/2022-01-05-runiverse-renv/index.md:23]:
```suggestion
The [latest version of renv (0.15.0)](https://rstudio.github.io/renv/news/index.html#renv-0150) now supports restoring packages that were installed from r-universe. In this post we explain how this works, and why.
```

File: content/blog/2022-01-05-runiverse-renv/index.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+A fundamental difference between [r-universe](https://r-universe.dev) and CRAN is that r-universe has *rolling* development model. The system automatically tracks the upstream git repositories and continuously builds and deploys from the latest commits, sometimes many times per day per package. We do not archive all of these versions in r-universe, but instead rely on the upstream git repositories to provide historical sources.
+
+Because packages published in r-universe are by definition built from a public git server, R and renv can treat such packages the same way as packages installed directly from git using e.g. `install_github()`. Effectively, r-universe serves a gateway to content on git, and if we know the upstream git-url and commit-id, it is unimportant how the package was installed exactly.
```
Comment (maelle) [inline content/blog/2022-01-05-runiverse-renv/index.md:31]: Should there be some pointers/ a footnote for readers who might want to make their project independent from any git resources, maybe https://rstudio.github.io/renv/reference/isolate.html & more generally reading up on renv caching etc.

**Review by maelle — COMMENTED** [review]
Nice :sparkles: I only have a few comments/suggestions.

File: content/blog/2022-01-05-runiverse-renv/index.md (hunk 3 above)
Comment (jeroen) [reply content/blog/2022-01-05-runiverse-renv/index.md:33]: I think that is out of scope for the article. The only point I want to make is that renv can re-install r-universe packages directly from GitHub. We reference renv docs in a few places for folks that want to dive deeper into renv usage.

**Pull request merged**